docs/upgrading.rst

```diff
@@ -62,7 +62,7 @@ Upgrading to 3.0
 Python compatibility
 ********************
 
-The marshmallow 3.x series supports Python >= 3.8.
+The marshmallow 3.x series require Python 3.
```

Comment:
```suggestion
The marshmallow 3.x series requires Python 3.
```

Reply:
Oh right. Series is singular here...

Reply:
Shouldn't I remove the whole Python 3 mention, in 2024?

Reply:
Oh, no, that's the upgrading to 3.0.0 section. Leaving it for history.

However, I'm tempted to also remove the Requirements section of the README.

Reply:
>  remove the Requirements section of the README.

👍